conans/test/functional/generators/cmake_find_package_multi_test.py

```diff
@@ -139,6 +139,105 @@ def build(self):
         client.run("create .")
         self.assertIn("Printing using a external module!", client.out)
 
+    @parameterized.expand([(False,), (True,)])
+    def test_build_modules_alias_target(self, use_components):
```

Comment:
Same issue as the one with the `cmake_find_package` generator: 

**helloTarget-release.cmake**
```cmake
...
set(namespace_BUILD_MODULES_PATHS_RELEASE "/private/var/folders/fc/6mvcrc952dqcjfhl4c7c11ph0000gn/T/tmpf8fefhlrconans/path with spaces/data/hello/1.0/_/_/package/f83037eff23ab3a94190d7f3f7b37a2d6d522241/share/cmake/target-alias.cmake")
...
set(namespace_comp_BUILD_MODULES_PATHS_RELEASE "/private/var/folders/fc/6mvcrc952dqcjfhl4c7c11ph0000gn/T/tmpf8fefhlrconans/path with spaces/data/hello/1.0/_/_/package/f83037eff23ab3a94190d7f3f7b37a2d6d522241/share/cmake/target-alias.cmake")
...
```

I expect the build-module to appear listed only under the `namespace_comp` ones...

**hello-config.cmake**
```cmake
...
########## BUILD MODULES ####################################################################
#############################################################################################

########## GLOBAL BUILD MODULES #############################################################

foreach(_BUILD_MODULE_PATH ${namespace_BUILD_MODULES_PATHS_DEBUG})
    include(${_BUILD_MODULE_PATH})
endforeach()

foreach(_BUILD_MODULE_PATH ${namespace_BUILD_MODULES_PATHS_MINSIZEREL})
    include(${_BUILD_MODULE_PATH})
endforeach()

foreach(_BUILD_MODULE_PATH ${namespace_BUILD_MODULES_PATHS_RELWITHDEBINFO})
    include(${_BUILD_MODULE_PATH})
endforeach()

foreach(_BUILD_MODULE_PATH ${namespace_BUILD_MODULES_PATHS_RELEASE})
    include(${_BUILD_MODULE_PATH})
endforeach()
...
```

Build-modules per component are not included.